services/src/northbound/src/main/java/org/openkilda/northbound/controller/SwitchController.java

```diff
@@ -251,4 +254,24 @@ public DeleteMeterResult deleteMeter(@PathVariable(name = "switch_id") String sw
                                          @PathVariable(name = "meter_id") long meterId) {
         return switchService.deleteMeter(switchId, meterId);
     }
+    
+    /**
+     * Configure port.
+     *
+     * @param switchId the switch id
+     * @param portNo the port no
+     * @param portConfig the port configuration payload
+     * @return the response entity
+     */
+    @ApiOperation(value = "Configure port on the switch", response = PortDto.class)
+    @ApiResponse(code = 200, response = PortDto.class, message = "Operation is successful")
+    @PutMapping(value = "/switches/{switch_id}/port/{port_no}/config",
+            produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<PortDto> configurePort(
```

Comment:
There is no need to wrap response into ResponseEntity as our controller defined with RestController annotation

Reply:
Not needed. Removed